magefile.go

```diff
@@ -230,8 +233,10 @@ func getBuildPath(buildOS, buildInfra string) string {
 	formattedOS = strings.Replace(formattedOS, ".", "", 1)
 
 	formattedOS = strings.Replace(formattedOS, "redhat", "rhel", 1)
-	formattedOS = strings.Replace(formattedOS, "1804", "18", 1)
-	formattedOS = strings.Replace(formattedOS, "2004", "20", 1)
+	if buildInfra == aws {
+		formattedOS = strings.Replace(formattedOS, "1804", "18", 1)
+		formattedOS = strings.Replace(formattedOS, "2004", "20", 1)
+	}
```

Comment:
This is annoying we have different file names for azure and aws

Reply:
💯

Reply:
i changed the file names as a part of this PR